tox.ini

```
@@ -52,6 +51,7 @@ commands =
 [testenv:clean]
```

FYI tox4 has a `base` env setting now: https://tox.wiki/en/latest/config.html#base

Wonder if it's possible to reset it to "nothing" with an empty string:
```suggestion
[testenv:clean]
base =
```